docs/quick-start/download-and-install.txt

```diff
@@ -0,0 +1,114 @@
+.. _laravel-quick-start-download-and-install:
+
+====================
+Download and Install
+====================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: php framework, odm, code example
+
+Prerequisites
+-------------
+
+To create the Quick Start application, you need the following software
+installed in your development environment:
+
+- `PHP <https://www.php.net/downloads>`__
+- `Composer <https://getcomposer.org/doc/00-intro.md>`__
+- A terminal app and shell. For MacOS users, use Terminal or a similar app.
+  For Windows users, use PowerShell.
+
+Download and Install the Dependencies
+-------------------------------------
+
+Complete the following steps to install and add the {+odm-short+} dependencies
+to a Laravel web application.
+
+.. procedure::
+   :style: connected
+
+   .. step:: Install the {+php-extension+}
+
+      The {+odm-short+} requires the {+php-extension+} to manage MongoDB
+      connections and commands.
+      Follow the `Installing the MongoDB PHP Driver with PECL <https://www.php.net/manual/en/mongodb.installation.pecl.php>`__
+      guide to install {+php-extension+}.
+
+   .. step:: Install Laravel
+
+      Ensure that the version of Laravel you install is compatible with the
+      version of {+odm-short+}. 
+
+
+      Then run the following command to install
+      Laravel.
```

Comment:
S: to match the punctuation of the other instructions on this page

```suggestion
      Laravel:
```

Reply:
Thanks for spotting this -- looks like I didn't fill out all the info I intended to include.